packages/coil-extension/src/popup/components/TipAmountFeedback.tsx

```diff
@@ -76,13 +77,17 @@ export const TipAmountFeedback = () => {
     return null
   }
 
+  const formattedTipCreditAmountUsd = Number.isInteger(totalTipCreditAmountUsd)
+    ? totalTipCreditAmountUsd
+    : totalTipCreditAmountUsd.toFixed(2)
```

Comment:
Do we need to keep the toFixed(2) here?

Reply:
Ooh I see this lets us do the decimal later if we want to... should we add a comment about why this is kept here?

Reply:
I think the comment is a good idea, I just added a comment for this.

Reply:
Looks good, thx!